modules/custom-status/custom-status.php

```diff
@@ -1646,22 +1646,7 @@ public function fix_post_row_actions( $actions, $post ) {
 		if ( empty( $actions['view'] ) )
 			return $actions;
 
-		if ( 'page' == $post->post_type ) {
-			$args = array(
-					'page_id'    => $post->ID,
-				);
-		} else if ( 'post' == $post->post_type ) {
-			$args = array(
-					'p'          => $post->ID,
-				);
-		} else {
-			$args = array(
-					'p'          => $post->ID,
-					'post_type'  => $post->post_type,
-				);
-		}
-		$args['preview'] = 'true';
```

Comment:
Shall we preserve the `preview` arg for all the links, not only for the case `$post->post_type` is `'post'`? Looks like it might be safer. The `get_preview_link` function currently adds it in the above case, while here it's always used.

Reply:
Yes, I think that it's a good idea. I thought that I already let this arg to `preview` but I think I rushed with the commit and forgot to check this once again. I'll do the change shortly.